public/scripts/views/forms/oauth-wso2.js

```diff
@@ -0,0 +1,127 @@
+(function (window) {
```

Comment:
Could we reuse https://github.com/appwrite/appwrite/blob/cae077d05b587482e728dc7f1fc2e4050e6f572f/public/scripts/views/forms/oauth-custom.js instead of adding this component like we do for the Apple and Microsoft adapters?

Would also be great of you could share screenshots of the UI with the final outcome.